mix.exs

```diff
@@ -4,7 +4,7 @@ defmodule Koans.Mixfile do
   def project do
     [app: :elixir_koans,
      version: "0.0.1",
-     elixir: ">= 1.2.1 and <= 1.4.0",
+     elixir: ">= 1.2.1 and <= 1.4.1",
```

Comment:
I like the idea of `~>1.4` on the upper end, which I _think_ means "anything greater than or equal to `1.4` up to, but not including `2.0`. That seems like a good target for us to not have to tweak this every time.

Something else to consider, as the koans grow, we're likely to have examples for features of the language not found in earlier versions. So it's probably a good idea to have both the lower and upper bounds of the version range in CI just to try and ensure the examples are valid for folks using versions in the range.